Sources/NSString+NXOAuth2.m

```diff
@@ -61,7 +61,7 @@ - (NSString *)nxoauth2_URLEncodedString;
     return (__bridge_transfer NSString *) CFURLCreateStringByAddingPercentEscapes(kCFAllocatorDefault, //Allocator
                                                                                   (__bridge CFStringRef)self, //Original String
                                                                                   NULL, //Characters to leave unescaped
-                                                                                  CFSTR("!*'();:@&=+$,/?%#[]"), //Legal Characters to be escaped
+                                                                                  NULL,
```

Comment:
Can you explain this change? These characters need to be escaped here for URL encoding to be RFC compliant.